<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditObject</class>
 <widget class="QDialog" name="EditObject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="position">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Position</string>
   </property>
  </widget>
  <widget class="QLabel" name="object_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>761</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Object</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="rotation">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Rotation</string>
   </property>
  </widget>
  <widget class="QLabel" name="radius_label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Radius</string>
   </property>
  </widget>
  <widget class="QLabel" name="radius_label_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>180</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Radius 2</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="col_b">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>140</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="color_label">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>100</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </widget>
  <widget class="QLabel" name="specular_label">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>180</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Specular</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="col_g">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>140</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="col_r">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>140</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="brim_label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>210</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>M:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="brip">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>260</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="brip_label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>250</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>P:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="brim">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>220</y>
     <width>67</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="radius_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>220</y>
     <width>67</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="pos_z">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>67</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-9999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>9999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="pos_y">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>67</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-9999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>9999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="pos_x">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>67</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-9999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>9999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="rot_z">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>67</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-9999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>99999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="rot_x">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>67</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-9999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>99999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="rot_y">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>67</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-9999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>99999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="radius">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>140</y>
     <width>67</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="angle">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>300</y>
     <width>67</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="angle_label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>260</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Angle</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="reflection">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>360</y>
     <width>67</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="reflection_label">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>320</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Reflection</string>
   </property>
  </widget>
  <widget class="QLabel" name="refraction_label">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>410</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Refraction</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="refraction">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>450</y>
     <width>67</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add_object">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>590</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Object</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="norm_z">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>67</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-9999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>9999.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="normal">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Normal</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="norm_x">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>67</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-9999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>9999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="norm_y">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>67</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-9999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>9999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dir_z">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>390</y>
     <width>67</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-9999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>9999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dir_x">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>67</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-9999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>9999.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="normal_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Direction</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dir_y">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>390</y>
     <width>67</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-9999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>9999.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="import_tex">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Import texture</string>
   </property>
  </widget>
  <widget class="QLabel" name="tex_path">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>490</y>
     <width>631</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
